CONTRIBUTING.md

```diff
@@ -1 +1,7 @@
-Please see [Contribution Guidelines](https://developers.mattermost.com/contribute/getting-started/)
```

Comment:
I'm 5/5 on this link remaining as it includes instructions around getting started with contributing, including setting up, cloning repos and so on for web app, server, mobile etc. The link to https://mattermost.com/contribute/ should remain though, as it introduces other ways to contribute.